packages/core-plugin-api/src/extensions/extensions.tsx

```diff
@@ -256,6 +256,7 @@ export function createReactExtension<
       };
 
       attachComponentData(Result, 'core.plugin', plugin);
+      attachComponentData(Result, 'core.name', name);
```

Comment:
```suggestion
      attachComponentData(Result, 'core.extensionName', name);
```

Feeling we need to make this a bit more specific

Reply:
Changed to `core.extensionName`